userenvs/fedora33.json

```diff
@@ -0,0 +1,105 @@
+{
+  "workshop": {
+    "schema": {
+      "version": "2020.03.02"
+    }
+  },
+  "userenv": {
+    "name": "fedora33",
+    "label": "Fedora 33",
+    "origin": {
+      "image": "registry.fedoraproject.org/fedora",
+      "tag": "33"
+    },
+    "properties": {
+      "packages": {
+        "type": "rpm",
+        "manager": "dnf"
+      }
+    }
+  },
+  "requirements": [
+    {
+      "name": "python39",
+      "type": "distro",
+      "distro_info": {
+        "packages": [
+          "python39",
+          "python3-pip",
+          "nodejs"
```

Comment:
Based on the labeling (`python39`) this (`nodejs`) should probably go in it's own requirements section.